src/core/sign.ml

```diff
@@ -64,12 +71,15 @@ let current_sign () =
   in
   PathMap.find mp !loaded
 
+let create_sym : string -> term -> prop -> expo -> sym = fun name typ p e ->
+  let path = (current_sign()).sign_path in (*file_to_module (current_path ())?*)
+  { sym_name = name ; sym_type = ref typ ; sym_path = path
+    ; sym_def = ref None ; sym_impl = []; sym_rules = ref []
+    ; sym_prop = p ; sym_expo = e ; sym_tree = ref Tree_types.empty_dtree }
+
 (** [new_sym ()] creates a new (private definable) symbol. *)
 let new_sym : string -> term -> sym = fun name typ ->
```

Comment:
Thanks to remove this function and replace it by its definition where it is used.